color-highlighter.lua

```diff
@@ -0,0 +1,28 @@
+-- This is test file only. the structure helps
```

Comment:
it's a dummy file, only used to visualize colors with color-highlighter. like this:

![image](https://user-images.githubusercontent.com/24642451/150668690-31e6b2df-1bae-4234-8985-d226799b3ec6.png)